Framework/Kernel/inc/MantidKernel/GitHubApiHelper.h.in

```diff
@@ -0,0 +1,60 @@
+#ifndef MANTID_KERNEL_GitHubApiHelper_H_
+#define MANTID_KERNEL_GitHubApiHelper_H_
+
+#include "MantidKernel/InternetHelper.h"
+
+namespace Mantid {
+namespace Kernel {
+
+/** GitHubApiHelper : A helper class for supporting access to the github api
+through HTTP and HTTPS, inherits from the InternetHelper.
+This class automatically adds the authorization to a read only account.
+
+Copyright &copy; 2016 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
+National Laboratory & European Spallation Source
+
+This file is part of Mantid.
+
+Mantid is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+
+Mantid is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+File change history is stored at: <https://github.com/mantidproject/mantid>
+Code Documentation is available at: <http://doxygen.mantidproject.org>
+*/
+class MANTID_KERNEL_DLL GitHubApiHelper : public InternetHelper {
+public:
+  GitHubApiHelper();
+  GitHubApiHelper(const Kernel::ProxyInfo &proxy);
+  virtual ~GitHubApiHelper();
```

Comment:
This destructor doesn't do anything. If that's not going to change, it could be omitted and the compiler should take care of it.